site/src/store/modules/schedule.ts

```diff
@@ -38,18 +54,24 @@ export default class Schedule extends VuexModule {
   }
 
   @Mutation
-  switchCurrentCourseSet(p: { name: string }): void {
+  async switchCurrentCourseSet(p: { name: string }): Promise<void> {
```

Comment:
Asynchronous functions need to be actions, not mutations.

Reply:
Good catch, fixed